### PR TITLE
Migrate Docs from GitBook

### PR DESCRIPTION
**Pull request by cwarnermm**
This PR migrates the Jira plugin documentation https://mattermost.gitbook.io/plugin-jira/ currently hosted via GitBook to the GitHub repository README file.

Once this PR is merged, the GitBook-hosted content can be archived/removed.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,23 +1,440 @@
...
 
 This repository is licensed under the Apache 2.0 License, except for the [server/enterprise](server/enterprise) directory which is licensed under the [Mattermost Source Available License](LICENSE.enterprise). See [Mattermost Source Available License](https://docs.mattermost.com/overview/faq.html#mattermost-source-available-license) to learn more.
 
 ## Development
 
 Read our [development docs](https://mattermost.gitbook.io/plugin-jira/development/environment) for this project, as well as the [Developer Workflow](https://developers.mattermost.com/extend/plugins/developer-workflow/) and [Developer Setup](https://developers.mattermost.com/extend/plugins/developer-setup/) documentation for more information about developing and extending plugins.
+
+### Environment
+
+To contribute to the project see https://www.mattermost.org/contribute-to-mattermost.
```
Comment (wiersgallak) [inline readme.md:418]: Wonder if we should remove this? The link is not working and the contribution process maybe changing.

Comment (cwarnermm) [inline readme.md:418]:
```suggestion
```

File: readme.md [hunk 2]
```diff
@@ -1,23 +1,439 @@
...
+* `/jira instance unalias [alias-name]` - Remove an alias from an instance
+* `/jira instance list` - List installed Jira instances
+* `/jira instance v2 <jiraURL>` - Set the Jira instance to process \"v2\" webhooks and subscriptions (not prefixed with the instance ID)
+* `/jira stats` - Display usage statistics
+* `/jira webhook [--instance=<jiraURL>]` -  Show the Mattermost webhook to receive JQL queries
+* `/jira v2revert` - Revert to V2 jira plugin data model
+
+### Notification Management
+
+#### What are Notifications?
```
Comment (wiersgallak) [inline readme.md:167]:
```suggestion
#### What are notifications?
```

File: readme.md [hunk 3]
```diff
@@ -1,23 +1,439 @@
...
+
+#### What is a notification subscription?
+
+Mattermost users can set up rules that define when a particular event with certain criteria are met in Jira that trigger a notification is sent to a particular channel. These subscription rules can specify the `Jira Project`, `Event Type`, `Issue Type`, and can filter out issues with certain values.
+
+When a user is setting up a notification subscription they'll only see the projects and issue types they have access to within Jira. If they can't see a project in Jira it won't be displayed as an option for that particular user when they are trying to set up a subscription in Mattermost.
+
+An approximate JQL query is output as well. This is not guaranteed to be valid JQL and is only shown as a reference to what the query may look like if converted to JQL.
+
+#### Who can set up Notification Subscriptions for a channel?
```
Comment (wiersgallak) [inline readme.md:187]:
```suggestion
#### Who can set up notification subscriptions for a channel?
```

File: readme.md [hunk 4]
```diff
@@ -1,23 +1,439 @@
...
+Connecting an account between Mattermost and Jira is a key part of the installation process and requires the end-user to authenticate with Jira and allow access to their Jira account. All `create`, `view`, `assign`, and `transition` operations are done using the logged-in user's Jira access token.
+
+* You must be signed into Mattermost on the same browser you are using to sign into Jira during `connect`.
+* The domain end users sign into Mattermost with on that browser must match the SiteURL in `config.json`.
+
+## User Guide
+
+### Getting Started
+
+To get started with the Jira/Mattermost connector is easy.  You'll first need to connect your Jira account with your Mattermost account so the system can perform actions such as searching, viewing and creating Jira issues on your behalf.
```
Comment (wiersgallak) [inline readme.md:300]:
```suggestion
To get started with the Jira/Mattermost connector is easy. You'll first need to connect your Jira account with your Mattermost account so the system can perform actions such as searching, viewing and creating Jira issues on your behalf.
```

**Review by wiersgallak — APPROVED** [review]

**Pull request merged**